ca/crl.go

```diff
@@ -134,8 +144,13 @@ func (ci *crlImpl) GenerateCRL(stream grpc.BidiStreamingServer[capb.GenerateCRLR
 
 	crlBytes, err := issuer.IssueCRL(ci.profile, req)
 	if err != nil {
+		var pkcs11Error *pkcs11.Error
+		if errors.As(err, &pkcs11Error) {
+			ci.signErrorCount.WithLabelValues("HSM").Inc()
+		}
```

Comment:
Just saw that's almost exactly what you did in the next PR.

Reply:
Yeah, I specifically didn't do that because I knew I was going to clean all this up in the follow-up. And doing it this way is [analogous to the OCSP signer](https://github.com/letsencrypt/boulder/blob/3786401f107be28f550366bc88c4a3bfc99e9a8b/ca/ocsp.go#L130-L133).